patches/minecraft/net/minecraft/client/renderer/EntityRenderer.java.patch

```diff
@@ -1,14 +1,40 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/EntityRenderer.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/EntityRenderer.java
-@@ -208,6 +208,7 @@
+@@ -91,6 +91,7 @@
+     private final MapItemRenderer field_147709_v;
+     private int field_78529_t;
+     private Entity field_78528_u;
++    private AxisAlignedBB pointedEntityBox;
```

Comment:
Since I've modified the way raytracing is handled so that the bounding box is also part of the RayTraceResult, it only made sense for me to make that work with both blocks and entities, hence the changes both here and in Block.